src/styles/components/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: grid;
+  grid-auto-flow: column;
+  grid-auto-columns: 20px;
```

Comment:
The same, you don't need a grid here. For one-dimension layouts, flex is the best choice.